lib/push.js

```diff
@@ -49,49 +61,76 @@ module.exports = function (log, db) {
      * @promise
      */
     notifyUpdate: function notifyUpdate(uid, reason) {
-      var events = reasonToEvents[reason] || reasonToEvents.accountVerify
+      reason = reason || 'accountVerify'
+      return this.pushToDevices(uid, reason)
+    },
+
+    notifyNewDevice: function notifyNewDevice(uid, deviceName, curDeviceId) {
+      var data = new Buffer(JSON.stringify({
+        action: 'fxacounts:new-device',
+        data: {
+          deviceName: deviceName
+        }
```

Comment:
@mhammond @rfk @kitcambridge seems like @eoger wants to coordinate the format of this with everyone. Thoughts on action + data for the push `payload`?